### ISSUE TITLE
Warning: Passing field metadata as a keyword arg is deprecated

### ISSUE THREAD

**Issue by dianaclarke** [conbench/conbench#1]
```
$ pytest -vv conbench/tests/api/test_index.py 
conbench/tests/api/test_index.py::TestIndexList::test_unauthenticated <- conbench/tests/api/_asserts.py PASSED                                                                          [ 25%]
conbench/tests/api/test_index.py::TestIndexList::test_api_index PASSED                                                                                                                  [ 50%]
conbench/tests/api/test_index.py::TestAPI::test_unknown_api_endpoint PASSED                                                                                                             [ 75%]
conbench/tests/api/test_index.py::TestAPI::test_ping PASSED                                                                                                                             [100%]

../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
../../envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195
  /Users/diana/envs/qa/lib/python3.8/site-packages/marshmallow/fields.py:195: RemovedInMarshmallow4Warning: Passing field metadata as a keyword arg is deprecated. Use the explicit `metadata=...` argument instead.
    warnings.warn(
```

**Comment by jgehrcke** [comment]
Closing for now, let's revisit if it makes sense.

**Comment by austin3dickey** [comment]
Oh I fixed this in #424! 🙂
